lib/schemes/local.js

```diff
@@ -23,7 +23,17 @@ export default class LocalScheme {
   mounted () {
     if (this.options.tokenRequired) {
       const token = this.$auth.syncToken(this.name)
-      this._setToken(token)
+
+      if (token) {
+        // Set logged in status
+        this.$auth.setLoggedIn(true)
```

Comment:
LoggedIn should be true only when token is available and user is fetched. (Unless user endpoint is disabled). Reason is `<div v-if="$auth.loggedIn">{{ $auth.user.name }}>`